installing/installing_aws_upi/installing-aws-upi.adoc

```diff
@@ -0,0 +1,81 @@
+[id="installing-aws-upi"]
+= Installing a cluster on AWS using CloudFormation templates
+include::modules/common-attributes.adoc[]
+:context: install-aws-upi
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a
+cluster on Amazon Web Services (AWS) using infrastructure that you provide.
+
+The easiest way to create this infrastructure is to use the provided
+CloudFormation templates. You can modify the templates to customize your
+infrastructure or use the information that they contain to create AWS objects
+according to your company's policies.
+
+.Prerequisites
+
+* xref:../../installing/installing_aws/installing-aws-account.adoc[Configure an AWS account]
+to host the cluster.
+
+include::modules/installation-overview.adoc[leveloffset=+1]
+
+include::modules/installation-aws-upi-requirements.adoc[leveloffset=+1]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/installation-provide-credentials.adoc[leveloffset=+1]
+
+include::modules/installation-initializing.adoc[leveloffset=+1]
+
+include::modules/installation-aws-config-yaml.adoc[leveloffset=+2]
+
+include::modules/installation-configuration-parameters.adoc[leveloffset=+2]
+
+include::modules/installation-generate-ignition-configs.adoc[leveloffset=+1]
+
+include::modules/installation-creating-aws-vpc.adoc[leveloffset=+1]
+
+include::modules/installation-cloudformation-vpc.adoc[leveloffset=+2]
+
+include::modules/installation-creating-aws-dns.adoc[leveloffset=+1]
+
+include::modules/installation-cloudformation-dns.adoc[leveloffset=+2]
+
+include::modules/installation-creating-aws-security.adoc[leveloffset=+1]
+
+include::modules/installation-cloudformation-security.adoc[leveloffset=+2]
+
+include::modules/installation-aws-upi-rhcos-ami.adoc[leveloffset=+1]
+
+include::modules/installation-creating-aws-bootstrap.adoc[leveloffset=+1]
+
+include::modules/installation-cloudformation-bootstrap.adoc[leveloffset=+2]
+
+include::modules/installation-creating-aws-control-plane.adoc[leveloffset=+1]
+
+include::modules/installation-cloudformation-control-plane.adoc[leveloffset=+2]
+
+[id="installing-workers-aws-upi"]
+== Creating worker nodes
+
+You can either manually create worker nodes before you install the cluster or
+use a MachineSet to create worker nodes after the cluster deploys.
+
+[WARNING]
+====
+It is better to use a MachineSet to allow the cluster to create and manage
+worker nodes than to manually create them.
```

Comment:
Judgemental?  If we are going to claim this, we should back it up, e.g. by talking about how easy it is to let the cluster automatically roll out newer RHCOS or recover from lost machines or some such.